config/recurring.yml

```diff
@@ -2,3 +2,8 @@ remove_temporary_record_command:
   command: "TemporaryRecord.expired.delete_all"
   priority: 100
   schedule: at 1am every day
+
+sync_provider_accreditation_status:
+  class: "Providers::SyncAccreditationStatusJob"
+  priority: 100
+  schedule: every 5 minutes
```

Comment:
Not sure of interval.

The date is stored without timestamp, right?
Which means this should be once a day thing, or when ever a new/updated accreditation occurs

Reply:
yeah once a day would be sufficient.